src/pages/references/changelog.md

```diff
@@ -21,6 +21,24 @@ contributors:
 
 # Changelog
 
+## 2024-09-30
+
+- Added many new **Text APIs** for improved text management.
+  - New [`TextNode.fullContent`](./document-sandbox/document-apis/classes/TextNode.md#fullcontent) accessor: returns the [`TextContentModel`](./document-sandbox/document-apis/classes/TextContentModel.md) containing the complete text string and its styles. `fullContent.text` supersedes `TextNode.text`, now deprecated.
```

Comment:
...returns the `TextContentModel` containing the complete text string and its styles associated to the Text Flow (aka Threaded Text or Overflow Text).

Reply:
Am wondering if we should have a separate section for deprecated methods to highlight it.

Reply:
Added the new text, moved the deprecation in an independent section.